src/app/requests/loading.tsx

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import LoadingContainer from "@/components/LoadingContainer";
+
+function Loading() {
```

Comment:
I don't see this component being used anywhere

Reply:
It's called automatically by Next when the request page is loading